# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../udev/scsi_id.config.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

install scsi_id.config in etc/udev instead of etc/ - mnemoc

--- ./extras/scsi_id/Makefile.orig	2007-06-07 10:56:52.000000000 -0400
+++ ./extras/scsi_id/Makefile	2007-06-07 10:57:54.000000000 -0400
@@ -44,7 +44,7 @@
 	$(E) "  GENHDR  " $@
 	$(Q) echo "/* Generated by make. */" > $@
 	$(Q) echo \#define SCSI_ID_VERSION     \"$(SCSI_ID_VERSION)\" >> $@
-	$(Q) echo \#define SCSI_ID_CONFIG_FILE     \"$(etcdir)/scsi_id.config\" >> $@
+	$(Q) echo \#define SCSI_ID_CONFIG_FILE     \"$(configdir)/scsi_id.config\" >> $@
 
 # man pages
 %.8: %.xml
@@ -74,9 +74,9 @@
 .PHONY: uninstall-man
 
 install-config:
-	@ if [ ! -r $(DESTDIR)$(etcdir)/scsi_id.config ]; then \
-		echo $(INSTALL_DATA) -D ./scsi_id.config  $(DESTDIR)$(etcdir); \
-		$(INSTALL_DATA) -D ./scsi_id.config $(DESTDIR)$(etcdir)/scsi_id.config; \
+	@ if [ ! -r $(DESTDIR)$(configdir)/scsi_id.config ]; then \
+		echo $(INSTALL_DATA) -D ./scsi_id.config  $(DESTDIR)$(configdir); \
+		$(INSTALL_DATA) -D ./scsi_id.config $(DESTDIR)$(configdir)/scsi_id.config; \
 	fi
 .PHONY: install-config
 
